public/index.html

```diff
@@ -3,10 +3,15 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="style.css">
     <title>Football Teams</title>
   </head>
   <body>
-    <input type="text" name="countryname" />
-    <button type="button" class="search">search</button>
+    <form class="search" action="/" style="">
+      <input type="text" id="search" placeholder="Search.." name="search2">
```

Comment:
Why is this called 'search2'?

Reply:
@jema28  I change the name to be searchBox